released/packages/coq-ott/coq-ott.0.31/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+maintainer: "palmskog@gmail.com"
+
+homepage: "http://www.cl.cam.ac.uk/~pes20/ott/"
+dev-repo: "git+https://github.com/ott-lang/ott.git"
+bug-reports: "https://github.com/ott-lang/ott/issues"
+license: "BSD-3-Clause"
+
+synopsis: "Auxiliary Coq library for Ott, a tool for writing definitions of programming languages and calculi"
+description: """
+Ott takes as input a definition of a language syntax and semantics, in a concise
+and readable ASCII notation that is close to what one would write in informal
+mathematics. It can then generate a Coq version of the definition, which requires
+this library.
+"""
+
+build: [make "-j%{jobs}%" "-C" "coq"]
+install: [make "-C" "coq" "install"]
+depends: [
+  "coq" {>= "8.5" & < "8.12~"}
+]
```

Comment:
```suggestion
]
conflicts: [
  "ott" { != version }
]
```
I think we should follow `coq-menhirlib` and constrain so that Ott and library versions are not out of sync.

Reply:
OK 👌